src/rdkafka_admin.c

```diff
@@ -5244,6 +5245,18 @@ void rd_kafka_AlterConsumerGroupOffsets(
                 }
         }
 
+        /* Copy offsets list for checking duplicated */
```

Comment:
Add TODO like below here as well.

Reply:
There should be no KIP for alterConsumerGroupOffsets still, I add a comment more generic